doc/developers/advanced_installation.rst

```diff
@@ -44,6 +44,20 @@ basis to help users test bleeding edge features or bug fixes::
 Building from source
 =====================
 
+In the vast majority of cases, building scikit-learn for development purposes
+can be done with::
+
+    pip install numpy scipy cython pytest sphinx sphinx-gallery flake8
+
```

Comment:
Maybe you can split the requirements to contribute to the code and those to contribute to the doc

Reply:
no mention of conda?

Reply:
On `conda`, I had a very hard time make it work on my machine, cause I couldn't find which compiler packages to install on conda (archlinux has a gcc which is too new for conda's `ld`). I've been wondering if it makes sense to have a conda file in the repo.